test/test_proxy_tensor.py

```diff
@@ -0,0 +1,213 @@
+                for node in traced.graph.nodes
+            )
+        )
+
+    def test_mode_tracing_factory_function_default_behavior(self):
+        def f(x):
+            return x + torch.randn(x.shape)
+
+        traced = make_fx(f)(torch.randn(3))  # default behavior should not trace factory functions
+        self.assertFalse(
+            any(
+                isinstance(node.target, torch._ops.OpOverloadPacket) and node.target._qualified_op_name == 'aten::randn'
+                for node in traced.graph.nodes
+            )
+        )
+
+make_fx_failures = {
+    xfail('allclose'),
+    xfail('nn.functional.dropout'),
+    xfail('linalg.eigvals'),
+    xfail('nn.functional.max_pool1d', device_type='cpu'),  # precision problems?
+    xfail('randn_like'),  # randomness
+    xfail('rand_like'),  # randomness
+    xfail('randint_like'),  # randomness
+    skip('new_empty'),  # nondeterministic
+    skip('empty_like'),  # nondeterministic
+    skip('linalg.lstsq', 'grad_oriented'),  # flaky
+    xfail('normal', '', device_type='cpu'),
+    xfail('normal', 'number_mean', device_type='cpu'),
+    xfail('multinomial', device_type='cpu'),
+    xfail('nn.functional.feature_alpha_dropout', 'with_train', device_type='cpu'),
+    xfail('bernoulli', device_type='cpu'),
+    xfail('nn.functional.dropout2d', device_type='cpu'),
+    skip('nn.functional.max_unpool1d', '', device_type='cpu'),  # flaky
+    skip('nn.functional.max_unpool2d', '', device_type='cpu'),  # flaky
+    skip('nn.functional.max_unpool3d', '', device_type='cpu'),  # flaky
+    skip('empty'),  # nondeterministic
+    skip('linalg.lstsq'),  # flaky, probably just a precision issue
+    xfail('histogram'),
+    xfail('scatter'),
+    # data-dependent control flow
+    xfail('cov'),
+    xfail('istft'),
+    xfail('nanquantile'),
+    xfail('nn.functional.gaussian_nll_loss'),
+    xfail('quantile'),
+    xfail('tensor_split'),
+    # Seems like it's creating a sparse tensor that isn't captured by tensor.is_sparse
+    xfail('sparse.sampled_addmm'),
+}
+
+
+class TestProxyTensorOpInfo(TestCase):
+    @ops(op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestProxyTensorOpInfo', 'test_make_fx_exhaustive', make_fx_failures
+             )
+    def test_make_fx_exhaustive(self, device, dtype, op):
+
+        def f(args, kwargs):
+            return op.op(*args, **kwargs)
+        sample_inputs_itr = op.sample_inputs(device, dtype, requires_grad=False)
+        new_f = None
+        for sample_input in sample_inputs_itr:
+            args = [sample_input.input] + list(sample_input.args)
+            kwargs = sample_input.kwargs
+
+            new_f = make_fx(f)(args, kwargs)
+            for arg in args:
+                if isinstance(arg, torch.Tensor) and arg.dtype == torch.float:
+                    arg.uniform_(0, 1)
+            try:
+                old_out = f(args, kwargs)
+            except Exception:
+                continue
```

Comment:
seems unsafe

Reply:
It's meant as an universal check against inputs that aren't valid.